### PR TITLE
Improve RI & Saings Plan monitoring

### PR DESCRIPTION
**Pull request by michaelwittig**

### CODE REVIEW

File: marbot.yml [hunk 1]
```diff
@@ -572,6 +586,40 @@ Resources:
...
+          Address: !Ref Topic
+        Notification:
+          ComparisonOperator: 'LESS_THAN'
+          NotificationType: ACTUAL
+          Threshold: !Ref SavingsPlansCoverageThreshold
+          ThresholdType: PERCENTAGE
+      Budget:
+        TimeUnit: MONTHLY
+        BudgetType: SAVINGS_PLANS_COVERAGE
+  SavingsPlansUtilzzation:
```
Comment (andreaswittig) [inline marbot.yml:606]: `SavingsPlansUtilzation`

File: marbot.yml [hunk 2]
```diff
@@ -82,6 +84,18 @@ Parameters:
     Type: Number
     Default: -1
     MinValue: -1
+  SavingsPlansCoverageThreshold:
+    Description: 'Receive an alert, if your monthly Savings Plans coverage (in percents) is lower than this value (works in us-east-1 only; set to -1 to disable).'
+    Type: Number
+    Default: -1
+    MinValue: -1
+    MaxValue: 100
+  SavingsPlansUtilzzationThreshold:
```
Comment (andreaswittig) [inline marbot.yml:93]: `SavingsPlansUtilzationThreshold` fix typo with search&replace in both files.

File: marbot-reserved-instance.yml [hunk 3]
```diff
@@ -0,0 +1,404 @@
+---
```
Comment (andreaswittig) [inline marbot-reserved-instance.yml:1]: Why did you split this up into a standalone stack? Because not everyone is using RIs?

**Review by andreaswittig — APPROVED** [review]

File: marbot-reserved-instance.yml (hunk 3 above)
Comment (michaelwittig) [reply marbot-reserved-instance.yml:1]: yes. I guess Savings Plans will replace RIs?

**Pull request merged**